src/components/Datagrid/styles.ts

```diff
@@ -215,7 +215,7 @@ export function useDatagridTheme(
       borderColor: themeValues.colors.border.neutralHighlight,
       fontFamily: "'Inter var', sans-serif",
       baseFontStyle: themeValues.fontSize.bodySmall,
-      headerFontStyle: themeValues.fontSize.bodySmall,
+      headerFontStyle: `${themeValues.fontWeight.bodyStrongSmall} ${themeValues.fontSize.bodySmall}`,
```

Comment:
**Question:** Mixed font styles

I'm wondering if we don't mix font styles here. In our design system, we have matching `fontWeight` and `fontSize`, so we should be able to use `themeValues.fontWeightBodyStrongSmall themeValues.fontSize.bodyStrongSmall`